scripts/scil_compute_local_tracking_gpu.py

```diff
@@ -84,6 +92,8 @@ def main():
 
     assert_inputs_exist(parser, [args.in_odf, args.in_mask, args.in_seed])
     assert_outputs_exist(parser, args, args.out_tractogram)
+    if args.compress is not None:
```

Comment:
Is there a value that means no compression ?

Reply:
Currently defaults to no compression if the arg is not set. We could make it otherwise, compress by default and disable if compress==0. Preferences?

Reply:
Done in 7308d9c !